Frends.SMTP.SendEmail/Frends.Smtp.SendEmail/SendEmailTask.cs

```diff
@@ -12,7 +12,10 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Frends.SMTP.SendEmail.Definitions;
+using System.Runtime.CompilerServices;
+using System.Net.Security;
 
+[assembly: InternalsVisibleTo("Frends.SMTP.Tests")]

```

Comment:
You have this set in both the project and here on code level. Lets have only one of those. And this assembly wide attribute is better to be put to its own Assembly.cs file.

Reply:
I moved this to its own AssemblyInfo.cs file